lib/Backends/CPU/Transforms.cpp

```diff
@@ -93,14 +93,30 @@ bool CPUBackend::transformPostLowering(Function *F,
     }
     if (auto *MN = dyn_cast<MaxNode>(&node)) {
       if (auto *splat = dyn_cast<SplatNode>(MN->getLHS())) {
-        auto MSN = F->addNode(new CPUMaxSplatNode(MN->getName(), MN->getRHS(),
+        NodeValue MNInput = MN->getRHS();
```

Comment:
Thanks for fixing this.

is it possible to consolidate the code from both branches in one place?
Seem like it's pretty much the same, once you know which NodeValue is from Splat and which is the other one.

Reply:
also please sign the CLA above :)